zipline/data/loader.py

```diff
@@ -12,6 +12,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from __future__ import print_function
```

Comment:
https://github.com/quantopian/zipline/pull/841/files#diff-0ec9e6870d90568b4a83e0daa3a317b1L290

This happened to work because `print(stock)` is still valid in Py2.  But from git blame it's clear that the intention was for that to be using the print function, which I suspect was either accidentally omitted or removed at a later point.